README.md

```diff
@@ -16,13 +14,87 @@ And then execute:
 
     $ bundle install
 
-Or install it yourself as:
+## Usage
 
-    $ gem install view_component-form
+Add a `builder` param to your `form_for` of `form_with`:
 
-## Usage
+```diff
+- = form_for @user do |f|
++ = form_for @user, builder: ViewComponent::Form::Builder do |f|
```

Comment:
Maybe a reference to [`default_form_builder`](https://api.rubyonrails.org/classes/ActionController/FormBuilder.html#method-i-default_form_builder) could be nice?